book/graphs/other/ltg01.qmd

```diff
@@ -155,12 +160,13 @@ g4 <- g4 + theme(
   axis.title  = element_blank()
 )
 g4 <- g4 + scale_x_continuous(breaks = adlb$AVISIT, labels = adlb$AVISIT_txt)
-g4
+plot <- g4
+plot
 ```
 
 #### Adding Mean
 
-```{r 2_2_add_mean}
+```{r plot51, test = list(plot_v51 = "plot")}
 # Add the mean along with the 95% CI at every visit.
 mean_text <- "Mean +/- 95% CI"
```

Comment:
Snapshot fails because the `.rds` file cannot access `mean_text` which is internally defined.
```
! could not find function "mean_text"
```

Reply:
you might try to add that to the testing env and see what happens, i.e. `test = list(plot_v51 = "plot", mean_text = "mean_text")`

Reply:
Ah! That should work. Thanks.

Reply:
This did not work as the env will consist of the variables `plot_v1`, `plot_v2` ... `plot_v51`, and `mean_text` so when we try to run `print(data_snap[["plot_v51"]])` to check the snapshot the `mean_text` is still not accessible, moreover the test will try to check the `print(data_snap[["mean_text"]])` thinking it should be snapshotted. So, for now, the easy fix was to declare these internally created objects/functions inside the test helpers.